hail/python/hailtop/batch/batch_pool_executor.py

```diff
@@ -486,11 +486,13 @@ async def async_cancel(self):
             return False
         await self.batch.cancel()
         self.fetch_coro.cancel()
+        await asyncio.wait([self.fetch_coro])
```

Comment:
I think it can just be `await fetch_coro`, but maybe I'm wrong.

Reply:
No. Just calling `await self.fetch_coro` will raise a CancelledError exception. We just want to wait for the cancel to be completed.